packages/lesswrong/lib/utils/timeUtil.ts

```diff
@@ -29,9 +29,9 @@ export const TimeOverrideContext = React.createContext<TimeOverride | null>(null
 // (This isn't necessary inside of event handlers or in any context that
 // isn't a component or used by components.)
 export function useCurrentTime(): Date {
-  const time = useContext(TimeOverrideContext);
-  if (time?.currentTime) {
-    return time.currentTime;
+  const time = useContext(EnvironmentOverrideContext);
+  if (time?.renderedAt) {
+    return new Date(time.renderedAt);
```

Comment:
Ah shoot, I just meant changing the where the time variable is cast.

Reply:
And I really just mean, can you convince me that _if_ there was some subtle bug at the boundary, you would have noticed.

Reply:
I've checked that the behaviour of the environment override stuff still works as expected*.

Another case where you might think this could introduce a subtle bug is if there is a mismatch between the client and server version. I don't think that should be an issue because the change doesn't affect the server apart from when fetching an SSR, in which case the client would get the latest bundle at the same time.

Obviously can't be 100% sure there is no bug here but I think it's low risk.

*For cache friendly pages, shows absolute dates rather than relative initially. For all pages, uses the server time rather than the client time for the first render

Reply:
SGTM, thanks for spelling it out and sorry for delaying you.